examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,169 @@
+{
+"metadata": {
+    "@type": "http://bib.schema.org/Audiobook",
+    "identifier": "https://librivox.org/flatland-a-romance-of-many-dimensions-by-edwin-abbott-abbott/",
+    "title": "Flatland: A Romance of Many Dimensions",
+    "author": [{
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
```

Comment:
I understand why you included multiple languages in `sort_as`, but I'm not sure that's a good thing to have. I really don't know how apps could handle that well.